{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -52,7 +52,19 @@ data: install-deps-all
 clean:
 	find . -type f -name "*.py[co]" -delete
 	find . -type d -name "__pycache__" -delete
+	find ./notebooks -name '*.ipynb' -exec pipenv run nbstripout {} \;
 
+## Trains and saves the model
+model:
```

Comment:
would we need to add the `model`, `prediction` and `deployment` to the `.PHONY:` line at the top?